ui/src/app/zones/views/ZonesListTitle/ZonesListTitle.tsx

```diff
@@ -0,0 +1,21 @@
+import { Fragment } from "react";
+
+import { Tooltip, Icon } from "@canonical/react-components";
+
+const ZonesListTitle = (): JSX.Element => {
+  return (
+    <Fragment>
```

Comment:
You can avoid `<Fragment>` and the associated import above by using `<>` and `</>` instead.

Reply:
Will do - thanks!